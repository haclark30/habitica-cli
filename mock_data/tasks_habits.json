{
    "data": [
        {
            "type": "habit",
            "text": "test habit 1",
            "up": true,
            "down": false,
            "counterUp": 10
        },
        {
            "type": "habit",
            "text": "test habit 2",
            "up": true,
            "down": true,
            "counterUp": 5,
            "counterDown": 0
        },
        {
            "type": "habit",
            "text": "test habit 3",
            "up": false,
            "down": true,
            "counterUp": 5,
            "counterDown": 3
        }
    ]
}