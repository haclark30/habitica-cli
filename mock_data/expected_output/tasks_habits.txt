test habit 1
 --10 ↑
test habit 2
 --5 ↑
 --0 ↓
test habit 3
 --3 ↓
